docs/source/weights_loading.rst

```diff
@@ -76,7 +76,7 @@ The Lightning checkpoint also saves the hparams (hyperparams) passed into the Li
    class MyLightningModule(LightningModule):
 
        def __init__(self, hparams, *args, **kwargs):
-           self.hparams = hparams
+           self = hparams
```

Comment:
@awaelchli @Borda it's weird, i thought the .rst were not tracked? not sure why the PR shows diffs here

Reply:
they are tracked, they are part of the documentation

Reply:
only the auto generated ones are not tracked, they are in the api subfolder

Reply:
btw is self = hparams correct?

Reply:
@awaelchli just answered, only RST in API folder is generated to have clear separation from manual documentation...